src/translator/response_builder.h

```diff
@@ -49,9 +55,7 @@ class ResponseBuilder {
     buildTranslatedText(histories, response);
 
     // Should always be after buildTranslatedText
-    if (responseOptions_.qualityScores) {
-      buildQualityScores(histories, response);
-    }
+    buildQualityScores(histories, response);
```

Comment:
You need this flag, quality score computation should be optional.

Reply:
Commit c0c1bbc